jujugui/static/gui/src/app/components/entity-content-readme/entity-content-readme.js

```diff
@@ -104,9 +104,9 @@ YUI.add('entity-content-readme', function() {
 
     render: function() {
       return (
-        <div className="entity-content_readme">
-          <h2 id="readme">Readme</h2>
-          <div className="entity-content_readme-content"
+        <div className="entity-content__readme">
+          <h2 className="entity-content__header" id="readme">Readme</h2>
```

Comment:
I don't know. It was already there, so I'm a bit nervous about removing it. Ditto for the other ID.